docs/UI/GetSupportedLanguages/assets/GetSupportedLanguages.txt

```diff
@@ -1,44 +1,23 @@
 participant App
```

Comment:
App can be removed from this diagram, it has no access to this information

Reply:
@jacobkeeler please see updates in ba730b1